lib/ui/fake_flutter/diagnosticable.dart

```diff
@@ -7,6 +7,9 @@
 // with a couple of tweaks to support error levels that haven't yet landed in
 // the core Flutter repo but should land soon.
 part of 'fake_flutter.dart';
+// Copyright 2017 The Chromium Authors. All rights reserved.
```

Comment:
Is the dupe of the copyright comment intended?

Reply:
I think not useful.  Removing.